app/controllers/api/v0/tags_controller.rb

```diff
@@ -1,20 +1,24 @@
 module Api
   module V0
     class TagsController < ApplicationController
-      caches_action :index,
-                    cache_path: proc { |c| c.params.permit! },
-                    expires_in: 10.minutes
-      before_action :set_cache_control_headers, only: %i[onboarding] # essentially static content
       respond_to :json
 
+      before_action :set_cache_control_headers, only: %i[index onboarding]
+
       def index
-        @page = params[:page]
-        @tags = Tag.order(taggings_count: :desc).page(@page).per(10)
+        page = params[:page]
+        per_page = (params[:per_page] || 10).to_i
+        num = [per_page, 1000].min
```

Comment:
this way we can allow the client to decide how many tags they want per page, keeping `10` by default